<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>193</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>阿里巴巴普惠体 2.0 65 Medium</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>抽奖,但一些</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextBrowser" name="Text">
       <property name="font">
        <font>
         <family>阿里巴巴普惠体 2.0 65 Medium</family>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>32</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>阿里巴巴普惠体 2.0 65 Medium</family>
          </font>
         </property>
         <property name="text">
          <string>抽奖次数(不能大于1000)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Edit">
         <property name="font">
          <font>
           <family>阿里巴巴普惠体 2.0 65 Medium</family>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Start">
         <property name="minimumSize">
          <size>
           <width>241</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>阿里巴巴普惠体 2.0 65 Medium</family>
          </font>
         </property>
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Exit">
         <property name="minimumSize">
          <size>
           <width>241</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>阿里巴巴普惠体 2.0 65 Medium</family>
          </font>
         </property>
         <property name="text">
          <string>保存并退出</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="minimumSize">
          <size>
           <width>351</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>阿里巴巴普惠体 2.0 65 Medium</family>
          </font>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
